src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -31,4 +31,27 @@
         <Setter Property="ToolTipService.Placement" Value="Mouse"/>
     </Style>
 
+    <Style TargetType="ComboBox">
+        <Setter Property="Margin" Value="0,0,0,20"/>

```

Comment:
I think this is fine though? The `CommonResources` styling only applies if you merge the resource dictionaries. The vast majority of the controls in the Settings UI need this since we want consistent styling throughout.